packages/preact/src/__tests__/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { h } from "preact"
```

Comment:
where is this used? or does it need to load _something_ from that module?

Reply:
This is a requirement, it's a hint to the compiler that it should switch to JSX mode.